File: MovieApp/Models/Movie.cs
using System;
using System.Collections.Generic;

namespace MovieApp.Models
{
    public partial class Movie
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public int? ReleaseYear { get; set; }
        public int? Genre { get; set; }

        public virtual Genre? GenreNavigation { get; set; }
    }
}

File: MovieApp/ClientApp/src/app/Movie.ts
//The angular side models need to match the JSON data and not the C# model

export class Movie{
    id: number;
    title: string;
    releaseYear: number;
    genre: number;

    constructor(id: number, title: string, releaseYear: number, genre: number) {
        this.id = id;
        this.title = title;
        this.releaseYear = releaseYear;
        this.genre = genre;
    }
}